src/components/InfoPanel/InfoPanel.test.js

```diff
@@ -31,6 +31,22 @@ describe("Info Panel", () => {
     expect(wrapper.find("Topology").length).toBe(1);
   });
 
+  it("renders the expanded topology on click", () => {
+    const store = mockStore(dataDump);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={["/models/group-test"]}>
+          <TestRoute path="/models/*">
+            <InfoPanel />
+          </TestRoute>
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(wrapper.find("Modal").length).toBe(0);
+    wrapper.find("i.p-icon--expand").simulate("click");
```

Comment:
Same here

Reply:
Please, don't target mark-up as that could change and functionality should continue to work.

Reply:
done